community/rspamd/APKBUILD

```diff
@@ -37,7 +37,7 @@ prepare() {
 		sed -e "s/@name@/${worker%:*}/g" -e "s/@port@/${worker#*:}/g" \
 			"$srcdir"/worker.conf.in > conf/worker-${worker%:*}.conf
 	done
-	sed -ri -e 's/worker /&"rspamd_proxy" /' \
+	sed -ri -e 's/"proxy"/"rspamd_proxy" /' \
 		conf/worker-proxy.conf
 	sed "$(grep -n -m1 'worker {' conf/rspamd.conf|cut -d: -f1),\$d" \
```

Comment:
this grep doesn't work. Change search string to '^worker '...
like this:
sed "$(grep -n -m1 '^worker ' conf/rspamd.conf|cut -d: -f1),\$d" \